### PR TITLE
Add link for upgrading from 2.x to 3.x

### PR DESCRIPTION
**Pull request by harikt**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 First alpha release.
 
+[See migration guide for details upgrading from 2.x to 3.x](https://github.com/auraphp/Aura.Sql/blob/695b240f1bd64dd5ba457935d7c0337be1055be5/docs/upgrade.md)
```
Comment (harikt) [inline CHANGELOG.md:7]: I have kept the specific commit for I don't know what will happen if we have more versions. Will the branches get deleted at that point. May be 3.x helps for now , so all the updated changes will also be reflected.

**Comment by harikt** [comment]
:-) that was so fast... Never expected ;-) .

File: CHANGELOG.md (hunk 1 above)
Comment (pmjones) [reply CHANGELOG.md:7]: That's fine; I think it goes well with that particular changelog entry. Having said that, a "final" conversion guide should be part of the 3.0.0 stable release.

**Comment by pmjones** [comment]
Simple changes are easy to merge. :-)

File: CHANGELOG.md (hunk 1 above)
Comment (harikt) [reply CHANGELOG.md:7]: > Having said that, a "final" conversion guide should be part of the 3.0.0 stable release.

Yes, that may be good. I have been looking how to reduce the changes mentioned regarding parser etc. For it is internal. So may be user level changes may be documented.

**Pull request merged**